corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -410,9 +411,14 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             //do something
         },
         selectRowAction: function (e) {
+            let currentText = document.getElementById('multi-select-continue-btn').innerText;
             if (e.target.checked) {
+                document.getElementById('multi-select-continue-btn').innerText = currentText.replace(String(this.selectedCaseIdCount), String(this.selectedCaseIdCount + 1));
```

Comment:
I'd stick the count in its own span, because string replaces are sketchy, even when side effects are unlikely.

Reply:
ah that is a much better solution: [afd379f](https://github.com/dimagi/commcare-hq/pull/31430/commits/afd379f5ab2989a48a37dfa5139ae8c5d0602853)